enhancements/installer/gcp-customer-provided-subnets.md

```diff
@@ -0,0 +1,258 @@
+---
+title: (GCP) Allow Customer-Provisioned Subnets
```

Comment:
I touched up the AWS analog [here][1].  Now that #111 has landed, you probably want to update yours similarly.

[1]: https://github.com/openshift/enhancements/pull/111/files#diff-b6500fac59b210d84188c47c008c5b49L2-R18